code/gb/bb/Makefile

```diff
@@ -127,8 +127,7 @@ $(OUT):	$(objs)
 	cd objs ; $(GBLINK) $(objs) -o ../$(OUT)
 
 clean: res
-	rm -f $(OUT) bb.map objs/*
```

Comment:
I don't want a rm -rf $(OUT) because... rm -rf is dangerous and what if OUT is defined wrong?

Can you limit your contributions to what will make it build on other setups?